### ISSUE TITLE
Add configurable timer beep

### ISSUE THREAD

**Issue by krisgesling** [MycroftAI/mycroft-timer#90]
It would be great for people to be able to select from a few timer beep options, or even add their own sound file to use instead.

This should be configurable from the Skill settings.

**Comment by derekantrican** [comment]
Agreed. The "two-beep" that is currently used is much too quiet for me. I'll probably find a way to customize this locally, but I'd love to be able to customize this the "proper" way
